spec/rspec/core/rake_task_spec.rb

```diff
@@ -163,14 +163,14 @@ def silence_output(&block)
       it "removes the environment variables", :if => RUBY_VERSION >= '1.9.0', :unless => RSpec::Support::Ruby.jruby? do
         with_env_vars 'MY_ENV' => 'ABC' do
           if RSpec::Support::OS.windows?
-            essential_shell_variables = /\["ANSICON", "ANSICON_DEF", "HOME", "TMPDIR", "USER"\]/
+            essential_shell_variables = "[\"ANSICON\", \"ANSICON_DEF\", \"HOME\", \"TMPDIR\", \"USER\"]"
           else
-            essential_shell_variables = /\["PWD"(?:, "SHLVL")?(?:, "_")?(?:, "__CF_USER_TEXT_ENCODING")?\]/
+            essential_shell_variables = "[\"PWD\", \"SHLVL\", \"_\", \"__CF_USER_TEXT_ENCODING\"]"
           end
 
           expect {
             task.with_clean_environment = true
-            task.ruby_opts = '-e "puts \"Environment: #{ENV.keys}\""'
+            task.ruby_opts = '-e "print ENV.keys.sort.inspect"'
```

Comment:
Is there any reason for removing "Environment: "?

Reply:
This was just another style change. I thought the `Environment: ` part was unnecessary since it doesn't verify the behavior under test.

Reply:
I've added this back to try to minimize the changes here.